[Fundamentals] Refer Zaraz on Google Analytics page (#4999)
cloudflare/cloudflare-docs: content/fundamentals/get-started/reference/google-analytics.md

# Using Google Analytics with Cloudflare

> Using Cloudflare does not affect Google Analytics (GA) tracking if it's added to the website [in one of ways recommended by Google](https://support.google.com/analytics/answer/9304153#add-tag).

Reviewer: ```suggestion
Using Cloudflare does not affect Google Analytics (GA) tracking if it is added to the website [in one of ways recommended by Google](https://support.google.com/analytics/answer/9304153#add-tag).
```

To troubleshoot potential issues with Google Analytics, refer to [Common GA setup mistakes](https://support.google.com/analytics/answer/1009683).

{{</Aside>}}

## Zaraz

> As an alternative to the standard setup of Google Analytics with tag/snippet, Cloudflare offers a way to use Google Analytics with [Zaraz](https://developers.cloudflare.com/zaraz/). Zaraz is a solution that allows Google Analytics to collect data without its script loaded on the website. If GA is set up this way, then not all features may be available. 

Reviewer: ```suggestion
As an alternative to the standard setup of Google Analytics with tag/snippet, Cloudflare offers a way to use Google Analytics with [Zaraz](/zaraz/). Zaraz is a solution that allows Google Analytics to collect data without its script loaded on the website. If GA is set up this way, then not all features may be available. 
```

{{<Aside type="note">}}

> Details about features of Google Analytics that are unavailable with Zaraz can be found in [Zaraz FAQ](zaraz/faq/#tools)

Reviewer: ```suggestion
Details about features of Google Analytics that are unavailable with Zaraz can be found in [Zaraz FAQ](/zaraz/faq/#tools)
```

Author: I was about to ask what can I do to fix the check, but you got it faster :)

> Details about features of Google Analytics that are unavailable with Zaraz can be found in [Zaraz FAQ](https://developers.cloudflare.com/zaraz/faq/#tools)

Reviewer: ```suggestion
Details about features of Google Analytics that are unavailable with Zaraz can be found in [Zaraz FAQ](zaraz/faq/#tools)
```
